src/asagi_converter.py

```diff
@@ -7,8 +7,8 @@
 SELECTOR = """SELECT
     `num` AS `no`,
     '{board_shortname}' AS `board_shortname`,
-    (CASE WHEN 1=1 THEN 1 ELSE NULL END) AS `closed`,
     DATE_FORMAT(FROM_UNIXTIME(`timestamp`), "%m/%d/%y (%a) %H:%i:%S") AS `now`,
+    DATE_FORMAT(FROM_UNIXTIME(`timestamp_expired`), "%m/%d/%y (%a) %H:%i:%S") AS `deltime`,
```

Comment:
Would be better if it was `deleted_time`. I like verbose variables (e.g. board_shortname)